547.py

```diff
@@ -0,0 +1,46 @@
+# This problem is to find number of connected component in an un-directed graph.
+
+# BFS Iterative Solution
+class Solution:
+    def findCircleNum(self, isConnected: List[List[int]]) -> int:
+        N = len(isConnected)
+        visited = [0] * N
```

Comment:
Using `0` for false and `1` for true is very much a C-style programming. In Python we have `True` and `False` keywords so it's better to use them for clarity.